### PR TITLE
add asset imgs & refactor footer nav

### PR DESCRIPTION
**Pull request by TGlide**
## What does this PR do?

- Adds assets imgs to assets page
- Refactors Footer Nav to use accessible accordions
- Adds dynamic nav indication to assets page
- Add link to assets page in footer

### CODE REVIEW

File: src/routes/assets/+page.svelte [hunk 1]
```diff
@@ -1,648 +1,666 @@
...
+
+    $: progress = (function getProgress() {
+        const sections = Object.values(Section);
+        const index = sections.indexOf(selected);
+        return (index / (sections.length - 1)) * 100;
+    })();
+
+    const handleVisibility = (section: Section) => {
+        return (e: CustomEvent<boolean>) => {
+            console.log(selectedMap);
```
Comment (TorstenDittmann) [inline src/routes/assets/+page.svelte:45]: leftover

File: src/routes/assets/+page.svelte [hunk 2]
```diff
@@ -1,648 +1,666 @@
...
+    enum Section {
+        NAMING = 'Naming',
+        LOGOTYPE = 'Logotype',
+        LOGOMARK = 'Logomark',
+        COLORS = 'Brand colors',
+        VISUALS = 'Product visuals',
+        CONTACT = 'Contact us'
+    }
+
+    const getSectionId = (section: Section) => section.toLowerCase().replace(/\s/g, '-');
```
Comment (TorstenDittmann) [inline src/routes/assets/+page.svelte:22]: fyi, this part we probably wanna re-use for pages like `/privacy`

Comment (TGlide) [reply src/routes/assets/+page.svelte:24]: Agreed! I was thinking if these pages should use markdoc though?

File: scripts/transform-images.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import { existsSync } from 'fs';
```
Comment (TorstenDittmann) [inline scripts/transform-images.js:1]: why do we need this, instead of re-using the vite optimizer?

Comment (TGlide) [reply scripts/transform-images.js:1]: This is not for smaller image outputs, but rather, for the assets page, where there's the option to download the same image in multiple formats.

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**